java/test/jmri/jmrix/swing/SystemConnectionComboBoxTest.java

```diff
@@ -21,16 +21,19 @@ public void testCTor() {
         Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         SystemConnectionComboBox t = new SystemConnectionComboBox();
         Assert.assertNotNull("exists",t);
+        t.dispose();
     }
 
     // The minimal setup for log4J
     @Before
     public void setUp() {
         JUnitUtil.setUp();
+        jmri.util.JUnitUtil.resetInstanceManager();
```

Comment:
I know this usb already merged, but why are we resetting the InstanceManager twice (JUnitUtil.setUp() and .tearDown() already reset it)?